spiffe/tornjak
# Adding typescript agent list [Part-1]

- Setup Typescript Environment
- Add Typescript to redux (Reducers & Actions)
- Add Typescript to agent-list (not including helper files)

Test steps
- Download a fresh copy of the branch and create an image, make sure everything runs without errors
- Diff between main and branch and go through changes
- Add a workload selector for an agent successfully
- Add agent/ agents to a new cluster
- Create an entry for the agent
- Check if the agent is found in the Tornjak dashboard with the correct number of entries
